docs/get-started/getting-started-on-prem.md

```diff
@@ -51,17 +51,17 @@ CUDA and will not run on other versions. The jars use a maven classifier to keep
 
 - CUDA 10.1 => classifier cuda10-1
 - CUDA 10.2 => classifier cuda10-2
-- CUDA 11.0 => classifier cuda11-0
+- CUDA 11.0 => classifier cuda11
 
 For example, here is a sample version of the jars and cudf with CUDA 10.1 support:
-- cudf-0.15-SNAPSHOT-cuda10-1.jar
-- rapids-4-spark_2.12-0.1.0.jar
+- cudf-0.15-cuda10-1.jar
+- rapids-4-spark_2.12-0.2.0.jar
```

Comment:
I wonder if we want to add a sentence that says the jar works with both 3.0.0 and 3.0.1

Reply:
Pointed to supported versions page, so as not to create another place to update each release.